doc/fping.pod

```diff
@@ -140,6 +140,11 @@ Print usage message.
 
 Set the IP TTL field (time to live hops).
 
+=item B<--print-ttl>
+
+Displays the TTL value in the output, if TTL is not present, "(TTL unknown)" is returned.
```

Comment:
Every IP packet has a TTL (called *Hop Count* for IPv6, but the IPv4 TTL field is de facto a hop count anyway). The issue is that `fping` can only read it when using raw sockets, and only for IPv4. I would prefer the man page to description to reflect this. I have just created pull request #346 to adjust this for the `--print-tos` option. Please use something similar.

Reply:
I did the same thing you did on that PR. Do you think it looks good?

Reply:
Please see my reply to your changed PR.